charts/gardener/admission-local/charts/runtime/templates/deployment.yaml

```diff
@@ -33,6 +33,9 @@ spec:
         - --webhook-config-mode=url
         - --webhook-config-url={{ printf "%s.%s" (include "name" .) (.Release.Namespace) }}
         - --webhook-config-namespace={{ .Release.Namespace }}
```

Comment:
Can we drop this parameter? It's not used anymore.

Reply:
It is still used for certificate management ([ref](https://github.com/gardener/gardener/blob/eeacca52df0976f170927cc3eedea24ac9333ef5/extensions/pkg/webhook/cmd/options.go#L350)).

Reply:
🤯 You're right! Can we take the opportunity to amend the description of this parameter? It's not only used/required if type `service` is used.